src/branding/default/branding.desc

```diff
@@ -182,8 +182,8 @@ images:
 style:
    sidebarBackground:    "#292F34"
    sidebarText:          "#FFFFFF"
-   sidebarTextSelect:    "#292F34"
-   sidebarTextHighlight: "#D35400"
+   sidebarTextCurrent:    "#292F34"
+   SidebarBackgroundCurrent: "#D35400"
```

Comment:
The casing is inconsistent here.

Reply:
This is to make the same as for Branding.h https://github.com/KaOSx/calamares/commit/abfad0ed1ddde07a1531890f658e8fc881964b85#diff-ace2b49f072feb0bbaeea9f8e4197ffb50b3dbfe14a3327226bebef8e726ad43R79